src/restconfig/src/test/java/org/geoserver/rest/resources/ResourceTest.java

```diff
@@ -295,7 +295,6 @@ public void testMove() throws Exception {
     
     @Test
     public void testMoveDirectory() throws Exception {
-        put("/rest/resource/mydir/mynewdir?operation=move", "/mydir");
```

Comment:
As far as I can tell, this was intended to fail before, but works fine with the new Apache Commons FileUtils move/rename methods, causing the subsequent line to fail.

Hoping to get more details from @NielsCharlier